doc/whats_new/v1.1.rst

```diff
@@ -260,6 +260,19 @@ Changelog
   hole; when set to True, it returns the swiss-hole dataset. :pr:`21482` by
   :user:`Sebastian Pujalte <pujaltes>`.
 
+- |Enhancement| Adds optional arguments `n_retries` and `delay` to functions 
+  :func:`fetch_20newsgroups`,
```

Comment:
So this links properly:

```suggestion
  :func:`datasets.fetch_20newsgroups`,
```

Similar suggestion for the rest of the functions here.

Reply:
Updated as suggested :)